.travis.yml

```diff
@@ -12,8 +12,8 @@ rvm:
   - rbx-2
 
 before_install:
-    - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
-    - gem install bundler -v '< 2'
+  - find /home/travis/.rvm/rubies -wholename '*default/bundler-*.gemspec' -delete
```

Comment:
Uninstalling a default gem seems to be impossible to do in a neat way, so I followed [this suggestion](https://github.com/travis-ci/travis-ci/issues/8717#issuecomment-366500378) to blast it off.